ecommerce/regis/static/sass/base/_navbar.scss

```
@@ -0,0 +1,54 @@
+// Navbar styles ecommerce/ecommerce/static/sass/partials/components/_navbar.scss.
+
+$navbar-gray: #242E33;
+
+.navbar {
+  border-bottom: solid rgba($navbar-gray, 0.1) 1px;
```

```suggestion
  border-bottom: solid rgba($main-elements-color, 0.1) 1px;
```

Replace variable for the one in _variables.scss